downstream/modules/platform/ref-platform-non-inst-database-inventory.adoc

```diff
@@ -38,7 +39,7 @@ registry_password='<registry password>'
 # Automation Hub Configuration
 #
 automationhub_admin_password='<password>'
-automationhub_pg_host='database-01.acme.org'
+automationhub_pg_host='controller'
```

Comment:
```suggestion
automationhub_pg_host='tower.acme.org'
```
I think that tower.acme.org is more suitable because as you can see, this is the automationcontroller host in this inventory example:  https://github.com/RedHatInsights/red-hat-ansible-automation-platform-documentation/blame/main/downstream/modules/platform/ref-platform-non-inst-database-inventory.adoc#L19-L20